### PR TITLE
[MAINTENANCE] Promote V1 Checkpoint objects

### PR DESCRIPTION
**Pull request by cdkini**
Change any methods that are prefixed with `v1_` to become their v0.18 counterparts (see checkpoint, actions, and renderers).

- [x] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted - run `invoke lint` (uses `ruff format` + `ruff check`)
- [x] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: tests/render/test_microsoft_teams_renderer.py [hunk 1]
```diff
@@ -1,167 +1,43 @@
...
-    ValidationResultIdentifier,
-)
+from great_expectations.checkpoint.checkpoint import CheckpointResult
 from great_expectations.render.renderer import MicrosoftTeamsRenderer
 
-# module level markers
-pytestmark = pytest.mark.big
 
+@pytest.mark.unit
+def test_MicrosoftTeamsRenderer_render(v1_checkpoint_result: CheckpointResult):
```
Comment (cdkini) [inline tests/render/test_microsoft_teams_renderer.py:8]: Was missing this so just backfilling

Comment (billdirks) [reply tests/render/test_microsoft_teams_renderer.py:8]: Can we rename the `v1_checkpoint_result` fixture to `checkpoint_result`?

File: tests/render/test_renderer.py [hunk 2]
```diff
@@ -9,36 +9,6 @@
 from great_expectations.render.renderer.renderer import Renderer
 
 
-@pytest.mark.unit
```
Comment (billdirks) [inline tests/render/test_renderer.py:12]: How is this test related to the v1 renaming?

**Review by billdirks — COMMENTED** [review]
LGTM. I have 2 questions inline.

File: tests/render/test_renderer.py (hunk 2 above)
Comment (cdkini) [reply tests/render/test_renderer.py:12]: `v1_render` became `render` and these tests no longer worked. I read them over and I don't think they provide enough value so I fully deleted them

File: tests/render/test_microsoft_teams_renderer.py (hunk 1 above)
Comment (cdkini) [reply tests/render/test_microsoft_teams_renderer.py:8]: we unfortunately already have a fixture with that name - I haven't yet deleted the legacy checkpoint config

**Review by billdirks — APPROVED** [review]
LGTM
